pages/office-of-solutions/tech-portfolio.md

```diff
@@ -1,12 +1,11 @@
 ---
-title: TTS Technology Portfolio
+title: TTS Technology Operations
 redirect_from:
   - /infrastructure/
-  - /tech-portfolio/
+  - /tech-operations/
```

Comment:
```suggestion
  - /tech-portfolio/
  - /tech-operations/
```

I recommend we keep the old redirect for a while in case anyone has bookmarked it.